Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -44,6 +44,7 @@ $isPremiumStorage = $false
 
 $sourcePath = $sourcePath.Trim('"')
 $storageAccount = $storageAccount.Trim()
+$containerName = $containerName.Trim().ToLower()
```

Comment:
We process the input in the first place and then apply logic to them.
In the first part, we process the input and in line 105, we apply logic to them.
We can have them seperately.